.github/config/pr-autoflow.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
It's fine to leave these `pr-autoflow` related files in this PR, but the repo will still need to be registered in the codeops process otherwise it won't receive updates.

https://github.com/endjin/endjin-codeops/blob/main/repo-level-processes/config/live/endjin.yml